### PR TITLE
fix: EBS metrics overrides in RDS and metrics query. Fixes #119

### LINKED ISSUE #119: bug(aws-rds): `VolumeIdleTime` metric not available in `AWS/RDS` namespace
There are multiple overrides for `VolumeIdleTime` in the RDS dashboard, this metric is not present in the `AWS/RDS` namespace and also not being queried anywhere in the dashboard

https://github.com/monitoringartist/grafana-aws-cloudwatch-dashboards/blob/b6d68860300b8efcee46e1be21855dec21f71c07/aws-rds/aws-rds.json#L94-L97

### PR DESCRIPTION
**Pull request by rohankmr414**
Signed-off-by: Rohan Kumar <rohankmr414@gmail.com>

Fixes #119

### CODE REVIEW

File: aws-rds/aws-rds.json [hunk 1]
```diff
@@ -1767,7 +1724,7 @@
             "filter": ""
           },
           "matchExact": true,
-          "metricName": "OldestReplicationSlotLag",
```
Comment (jangaraj) [inline aws-rds/aws-rds.json:1770]: I would prefer to keep original metric here. ReplicationSlotDiskUsage can be additional panel, query.

File: aws-rds/aws-rds.json [hunk 2]
```diff
@@ -1801,7 +1758,7 @@
             "filter": ""
           },
           "matchExact": true,
-          "metricName": "ReplicaLag",
```
Comment (jangaraj) [inline aws-rds/aws-rds.json:1804]: I would prefer to keep original metric here. TransactionLogsDiskUsage can be additional panel, query.

File: aws-rds/aws-rds.json (hunk 1 above)
Comment (rohankmr414) [reply aws-rds/aws-rds.json:1770]: There's another panel with the original metric called `OldestReplicationSlotLag/ReplicaLag` this one's a duplicate panel which has title `ReplicationSlotDiskUsage/TransactionLogsDiskUsage`

File: aws-rds/aws-rds.json (hunk 2 above)
Comment (rohankmr414) [reply aws-rds/aws-rds.json:1804]: There's another panel with the original metric called `OldestReplicationSlotLag/ReplicaLag` this one's a duplicate panel which has title `ReplicationSlotDiskUsage/TransactionLogsDiskUsage`

**Comment by jangaraj** [comment]
Awesome! Thank you for your contribution Rohan 👍

**Pull request merged**